app/forms/hyrax/forms/permission_template_form.rb

```diff
@@ -73,8 +73,13 @@ def update(attributes)
       # Copy this access to the permissions of the Admin Set or Collection and to
       # the WorkflowResponsibilities of the active workflow if this is an Admin Set
       def update_access(remove_agent: false)
-        source_model.reset_access_controls!
-        update_workflow_responsibilities(remove_agent: remove_agent) if source_model.is_a?(AdminSet)
+        case source_model
+        when Valkyrie::Resource
+          source_model.permission_manager.acl.save
+        else
+          source_model.reset_access_controls!
```

Comment:
isn't there an existing pattern for replacing `reset_access_controls!`?

i think it might be `reset_access_controls_for(collection: source)`, established in db58e27e787f87dd930d8ac7b04e28741712d90c

Reply:
Switched to this pattern.